src/Microsoft.DotNet.CodeAnalysis.net45/Microsoft.DotNet.CodeAnalysis.net45.csproj

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
```

Comment:
I believe we are for the net45 projects. For netcore projects I agree where we are not copying dependencies, but for net45 we will copy down dependencies so I do believe the property should stay.